Sprint-1/implement/dedupe.js

```diff
@@ -1 +1,20 @@
-function dedupe() {}
+function dedupe(list) {// return [] if list is empty array
+  if (list.length === 0){
+    return [];
+  } 
+  // declares an empty array to push unique value
+ let uniqueList = [];
+ return list.filter ((item) => {
+  if (uniqueList.includes(item)){
+  return false;
+ }
+uniqueList.push(item);
+return true;
+});
```

Comment:
This block of code is not properly indented, making it hard to read. In VSCode, you can right-click the code in the editor and select "Format Document" to format your code.